runtime/syntax/rsc.vim

```diff
@@ -11,6 +11,8 @@ endif
 
 syn case ignore
 
+syn iskeyword @,-
```

Comment:
You've missed digits used in, for example, `toip6`.  So at a minimum I think you'll need:
```vim
syn iskeyword @,48-57,-
```

Reply:
You're right, fixed.